views/scss/qtiCreator/_outcome-editor.scss

```diff
@@ -29,6 +29,7 @@
             display: inline-block;
             margin: 0;
             width: calc(100% - 44px);
+            overflow: hidden;
```

Comment:
**thought:** This looks a little rough. But it does the job.
**suggestion:** Could you add an ellipsis to ease the display for the user?

Reply:
yeah with ellipsis it looks much better